rdrf/registry/groups/models.py

```diff
@@ -152,8 +155,8 @@ def notices(self):
     def in_registry(self, registry_model):
         return self.registry.filter(pk=registry_model.pk).exists()
 
-    def in_group(self, name):
-        return self.groups.filter(name__icontains=name).exists()
+    def in_group(self, *names):
+        return self.groups.filter(reduce(operator.or_, ((Q(name__icontains=name) for name in names)))).exists()
```

Comment:
Any idea why this filters with an `icontains`, is there any instances where the saved Group name is anything other than the `RDRF_GROUP` value?

It would be nice to see a unit test for this one :)

Reply:
I think it's just to allow you to set the group name case-insensitively in the admin interface, so you don't create a group called 'Parents' and wonder why it's not working.
Good point on adding a test.

Reply:
There is an `iexact` that might be a bit more accurate?
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#iexact

I don't know if it's worth changing though.

Reply:
Yeah that's probably better, I'll have a look at what it might break.